.clang-format.chromium

```diff
@@ -0,0 +1,60 @@
+# Defines the ANGLE style for automatic reformatting.
+# https://code.google.com/p/angleproject/wiki/CodingStandard
+# See Clang docs: http://clang.llvm.org/docs/ClangFormatStyleOptions.html
+BasedOnStyle: Chromium
+
+# Allow double brackets such as std::vector<std::vector<int>>.
+Standard: Cpp11
+
+# Indent 4 spaces at a time.
+IndentWidth: 4
+
+# Keep lines under 100 columns long.
+ColumnLimit: 100
+
+# Always break before braces
+BreakBeforeBraces: Custom
+BraceWrapping:
+#  TODO(lujc) wait for clang-format-9 support in Chromium tools
+#  AfterCaseLabel: true
+  AfterClass: true
```

Comment:
```suggestion
AfterClass: true
```

Reply:
This one was a verbatim copy of the file from Chromium tree.

Reply:
I kept it as-is in Chromium mainline, renamed this to .clang-format